Update permissions-microsoft-365-security-center.md (#10278)
MicrosoftDocs/microsoft-365-docs: microsoft-365/security/office-365-security/permissions-microsoft-365-security-center.md

---

> title: Microsoft Defender for Office 365 Permissions in the Microsoft 365 Defender portal

Reviewer: ```suggestion
title: Microsoft Defender for Office 365 permissions in the Microsoft 365 Defender portal
```

[!INCLUDE [MDO Trial banner](../includes/mdo-trial-banner.md)]

**Applies to**
- [Microsoft Defender for Office 365 plan 1 and plan 2](defender-for-office-365.md)
- [Exchange Online Protection](exchange-online-protection-overview.md)
- [Microsoft 365 Defender](../defender/microsoft-365-defender.md)

> Access to Microsoft Defender for Office 365 capabilities in the Microsoft 365 Defender portal can be managed collectively by using Global roles in Azure Active Directory (AAD). If you need greater flexibility and control over access to specific Microsoft Defender for Office 365 product data, access can also be managed with the creation of Custom roles.

Reviewer: ```suggestion
Access to Microsoft Defender for Office 365 capabilities in the Microsoft 365 Defender portal can be managed collectively by using global roles in Azure Active Directory (Azure AD). If you need greater flexibility and control over access to specific Microsoft Defender for Office 365 product data, access can also be managed with the creation of custom roles.
```

> To manage permissions in the Microsoft 365 Defender portal, go to **Permissions & roles** > ** Email & collaboration roles** or <https://security.microsoft.com/securitypermissions>. You need to be a **global administrator** or a member of the **Organization Management** role group in the Microsoft 365 Defender portal. Specifically, the **Role Management** role allows users to view, create, and modify role groups in the Microsoft 365 Defender portal, and by default, that role is assigned only to the **Organization Management** role group.

Reviewer: ```suggestion
To manage permissions in the Microsoft 365 Defender portal, go to **Permissions & roles** > **Email & collaboration roles** or <https://security.microsoft.com/securitypermissions>. You need to be a **Global administrator** or a member of the **Organization Management** role group in the Microsoft 365 Defender portal. Specifically, the **Role Management** role allows users to view, create, and modify role groups in the Microsoft 365 Defender portal, and by default, that role is assigned only to the **Organization Management** role group.
```

> [!NOTE]

> > Some capabilities of Microsoft Defender for Office requrie additional permissions in the Exchange Admin Center, see [Permissions in Exchange Online](/exchange/permissions-exo/permissions-exo)

Reviewer: ```suggestion
> Some capabilities of Microsoft Defender for Office require additional permissions in the Exchange admin center. For more information, see [Permissions in Exchange Online](/exchange/permissions-exo/permissions-exo).
```

:::image type="content" source="../../media/2a16d200-968c-4755-98ec-f1862d58cb8b.png" alt-text="The relationship of a role group to its roles and members" lightbox="../../media/2a16d200-968c-4755-98ec-f1862d58cb8b.png":::

> ## Roles and role groups in the Microsoft 365 Defender portal to Access Microsoft Defender for Office 365 Data and Experiences

Reviewer: ```suggestion
## Roles and role groups in the Microsoft 365 Defender portal to access Microsoft Defender for Office 365 data and experiences
```

:::image type="content" source="../../media/permissions-manage-in-azure-ad-link.png" alt-text="The link to manage permissions in Azure Active Directory" lightbox="../../media/permissions-manage-in-azure-ad-link.png":::

> For more information, see [View and assign administrator roles in Azure Active Directory](/azure/active-directory/users-groups-roles/directory-manage-roles-portal) and and [Manage access to Microsoft 365 Defender with Azure Active Directory global roles](/microsoft-365/security/defender/m365d-permissions).

Reviewer: ```suggestion
For more information, see [View and assign administrator roles in Azure Active Directory](/azure/active-directory/users-groups-roles/directory-manage-roles-portal) and [Manage access to Microsoft 365 Defender with Azure Active Directory global roles](/microsoft-365/security/defender/m365d-permissions).
```
